src/plugins_3rdparty/primer3/src/Primer3Dialog.cpp

```diff
@@ -595,172 +747,280 @@ void Primer3Dialog::sl_saveSettings() {
         fileName += ".txt";
     }
 
-    QSettings diagSettings(fileName, QSettings::IniFormat);
+    QFile file(fileName);
+    if (file.exists()) {
+        file.remove();
```

Comment:
It was needed because in case of file exists new parameters started to be written to the end of file, but I found out a better solution - `QIODevice::Truncate` to the open mode

Reply:
Yes I meant that flag.